html/web_basics/index.html

```diff
@@ -0,0 +1,8 @@
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+	<h1>Hello World :D</h1>
```

Comment:
Hey can you fix this spacing? You probably need to use sublime since Nano doesn't space properly.